pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/PulsarWorkerService.java

```diff
@@ -389,7 +397,11 @@ private static Policies createFunctionsNamespacePolicies(String pulsarFunctionsC
 
     private void tryCreateNonPartitionedTopic(final String topic) throws PulsarAdminException {
         try {
-            getBrokerAdmin().topics().createNonPartitionedTopic(topic);
+            TopicName topicName = TopicName.get(topic);
+            List<String> topics = getBrokerAdmin().topics().getList(topicName.getNamespace());
+            if (topics.stream().noneMatch(n -> TopicName.get(n).equals(topicName))) {
+                getBrokerAdmin().topics().createNonPartitionedTopic(topic);
+            }
         } catch (PulsarAdminException e) {
             if (e instanceof PulsarAdminException.ConflictException) {
                 log.warn("Failed to create topic '{}': {}", topic, e.getMessage());
```

Comment:
this is the only line you want to change.
you can switch this to DEBUG

Reply:
Switch to the "debug" level is easier, but you can still see some logs in the broker.